marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/mapping.sjs

```diff
@@ -24,7 +24,7 @@ const hubEs = require("/data-hub/5/impl/hub-es.sjs");
 const collections = ['http://marklogic.com/data-hub/mappings', 'http://marklogic.com/data-hub/steps/mapping', 'http://marklogic.com/data-hub/steps'];
 const databases = [dataHub.config.STAGINGDATABASE, dataHub.config.FINALDATABASE];
 const permissions = [xdmp.permission(dataHub.consts.DATA_HUB_MAPPING_WRITE_ROLE, 'update'), xdmp.permission(dataHub.consts.DATA_HUB_MAPPING_READ_ROLE, 'read')];
-const requiredProperties = ['name', 'targetEntityType', 'selectedSource'];
+const requiredProperties = ['name'];
```

Comment:
I think targetEntityType and selectedSource are supposed to stay here - did you mean to remove them? They don't need to exist in matching/merging yet.

Reply:
Re-added